core/serialize/serialize.cc

```diff
@@ -929,7 +929,7 @@ void SerializerImpl::pickle(Pickler &p, const ast::TreePtr &what) {
     typecase(
         what.get(),
         [&](ast::Send *s) {
-            pickleAstHeader(p, 2, s);
+            pickleAstHeader(p, 2, s->loc);
```

Comment:
Maybe in a follow-up it'd make sense to use the enum value rather than these hardcoded constants.

Reply:
This is a great idea -- I'll do it in a follow-up in the interest of finally merging this thing.